crm_lead_address_street3/__manifest__.py

```diff
@@ -29,8 +29,8 @@
     'website': 'http://www.camptocamp.com',
     'data': ['view/crm_lead_view.xml'],
     'demo': [],
-    'test': ['test/test_street3.yml'],
-    'installable': False,
+    'test': [],
```

Comment:
it's not needed as it will be automatically inspected (see https://www.odoo.com/documentation/10.0/reference/testing.html as reference).